_data/other.yml

```diff
@@ -212,6 +212,13 @@ websites:
       - software
     doc: https://support.everlaw.com/hc/en-us/articles/206312165
 
+  - name: FAX.PLUS
+    url: https://www.fax.plus
+    img: faxplus.png
+    tfa:
+      - software
```

Comment:
```suggestion
      - totp
```

Reply:
Resolved with 94e0792. Why didn't I see that 😐